### ISSUE TITLE
Does the dataset of Gradio support selected events

### ISSUE THREAD

**Issue by guoshan333211** [gradio-app/gradio#4480]
- [ ] I have searched to see if a similar issue already exists.

**Is your feature request related to a problem? Please describe.**  

**Describe the solution you'd like**  

**Additional context**

**Comment by abidlabs** [comment]
Yes, it supports a `.click()` event. By the way, for general questions like this (that are not feature requests or bug reports), please ask in our [Forums](https://discuss.huggingface.co/c/gradio/26) or [our Discord](http://discord.gg/feTf9x3ZSB). (I'll close this issue)
